cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -114,6 +114,12 @@
   }
 }
 
+.a-card-institution-name {
+  .heading-5( @text-shadow: @gray );
+  color: @gray;
+  margin-bottom: 0;
+}
+
```

Comment:
You could delete this class and just use a couple utility classes on the institution name div (`class="h5 u-mb0"`) but the utility classes wouldn't let us control the text color so nvm.